.github/workflows/build-and-run-model.yaml

```diff
@@ -26,12 +27,11 @@ jobs:
       # required in order to allow the reusable called workflow to push to
       # GitHub Container Registry
       packages: write
-    uses: ./.github/workflows/build-and-run-batch-job.yaml
+    uses: ccao-data/actions/.github/workflows/build-and-run-batch-job.yaml@master
```

Comment:
While it would be better to pin to a specific release of `ccao-data/actions`, my thinking is that it makes sense to pin to `master` while we're iterating quickly, and then pin to a tag once things are feeling stable.

Alternatively, we could pin to a specific commit SHA instead, but we won't know what that SHA is until https://github.com/ccao-data/actions/pull/1 gets merged.